.travis.yml

```diff
@@ -97,6 +97,9 @@ matrix:
       rust: beta
 
   allow_failures:
+    # There are too many inconsistent failures on this arch to make it Tier 1.
```

Comment:
I agree, allowing failures would also make this Tier 2 not Tier 3, so good catch!